packages/devtools_app/lib/src/flutter/screen.dart

```diff
@@ -18,14 +18,38 @@ import 'scaffold.dart';
 /// Defines pages shown in the tabbar of the app.
 @immutable
 abstract class Screen {
-  const Screen(this.type);
+  const Screen(this.type, this.title, this.iconData, {this.tabKey});
```

Comment:
to be consistent with flutter consider renaming `iconData` to `icon` and make it a named parameter.

Reply:
also make title a named parameter

Reply:
will do